inf-clojure.el

```diff
@@ -236,7 +237,15 @@ Clojure to load that file."
 (define-obsolete-variable-alias 'inf-clojure-load-command 'inf-clojure-load-form "2.0.0")
 
 (defcustom inf-clojure-load-form-lumo "(clojure.core/load-file \"%s\")"
-  "Format-string for building a Clojure expression to load a file.
+  "Planck format-string for building a Clojure expression to load a file.
```

Comment:
Maybe little typo here, shouldn't it be for Lumo? I have probably forgot to update this one myself :smile: